src/components/ProfileMatch/styles.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+export const IconDiv = styled.div`
+  min-width: 16px;
+  min-height: 16px;
+  text-align: center;
+  align-items: center;
+`;
+
+export const IconGroup = styled.div`
+  display: flex;
+  direction: row;
+  align-items: center;
+  gap: 16px;
+`;
```

Comment:
should we replace this with just the `Flex` component?

Reply:
I thought that the named component would make the code more clear, but I'm down to change it to a `Flex` component.

Reply:
yeah let's go ahead an update to `Flex`